Mage/src/main/java/mage/abilities/effects/common/DamageTargetEffect.java

```diff
@@ -57,25 +60,47 @@ public DamageTargetEffect(DynamicValue amount) {
         this(amount, true);
     }
 
+    public DamageTargetEffect(DynamicValue amount, DynamicValue.EffectPhrasing phrasing) {
+        this(amount, true, phrasing);
+    }
+
     public DamageTargetEffect(DynamicValue amount, String whoDealDamageName) {
         this(amount, true);
         this.sourceName = whoDealDamageName;
     }
 
+    public DamageTargetEffect(DynamicValue amount, String whoDealDamageName, DynamicValue.EffectPhrasing phrasing) {
```

Comment:
I recommend to use `withPhrasing()` logic instead duplicate methods for each constructor.